apps/server/src/modules/deletion/repo/deletion-request.repo.spec.ts

```diff
@@ -183,14 +183,14 @@ describe(DeletionRequestRepo.name, () => {
 
 				const results = await repo.findAllItemsToExecution();
 
-				expect(results.length).toEqual(3);
+				expect(results.length).toEqual(1);
 
 				// Verify explicit fields.
 				expect(results).toEqual(
 					expect.arrayContaining([
 						expect.objectContaining(expectedArray[0]),
-						expect.objectContaining(expectedArray[1]),
-						expect.objectContaining(expectedArray[2]),
+						// expect.objectContaining(expectedArray[1]),
+						// expect.objectContaining(expectedArray[2]),
```

Comment:
Same here - delete commented out lines if not needed in the end.

Reply:
done